canonical-web-and-design/anbox-cloud.io
# Use image_template to defer offscreen images

## Done

- Set up the [image_template script](https://github.com/canonical-web-and-design/canonicalwebteam.image-template).
- Replaced all <img> elements in index.html with the script.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure the rendered page looks the same
- Ensure that all the attributes of the old <img> elements are correctly represented in the scripts.

## Issue / Card

Fixes #28
